src/persist-client/src/internal/state.rs

```diff
@@ -769,19 +769,7 @@ where
 
         let existed = self.leased_readers.remove(reader_id).is_some();
         if existed {
-            // TODO: Re-enable this once we have #15511.
-            //
-            // Temporarily disabling this because we think it might be the cause
-            // of the remap since bug. Specifically, a clusterd process has a
-            // ReadHandle for maintaining the once and one inside a Listen. If
-            // we crash and stay down for longer than the read lease duration,
-            // it's possible that an expiry of them both in quick succession
-            // jumps the since forward to the Listen one.
-            //
-            // Don't forget to update the downgrade_since when this gets
-            // switched back on.
-            //
-            // self.update_since();
+            self.update_since();
```

Comment:
This PR is a lovely idea, but unfortunately persist already has its hands full with support issues and this is a pretty foundational bit of code. Given that there isn't an immediate customer need here, I'd prefer to err on the side of caution and leave things as they were.

Thanks!

Reply:
@def-: Can we keep a list of (temporarily) ignored issues somewhere? Then, we also don't need to remove unresolved issue references to be able to merge this PR.

Reply:
I've created https://github.com/MaterializeInc/materialize/issues/22789 for re-enabling this, in case you just want to switch the issue ID over to an new issue and keep the tooling as-is!